### DISCUSSION TITLE
使用远程api上传图片失败

### DISCUSSION THREAD

**Discussion by GriMu** [Qexo/Qexo#380] [Q&A] [answered]
1.使用了vercel部署了qexo
2.在配置图床时选择的是自己部署的兰空图床
3.已经确认了兰空图床通过postman正常山上传图片
![image](https://github.com/Qexo/Qexo/assets/38001542/28063686-165f-4afb-a863-37f8516e590a)

4.在qexo中上传提示错误信息如下，能帮忙看下是哪里配置不对吗
![image](https://github.com/Qexo/Qexo/assets/38001542/8f0d2e80-12b8-4152-88a5-7fc46039d94c)
![image](https://github.com/Qexo/Qexo/assets/38001542/49a44d21-b5fb-43ea-b315-50a7fb8ad60d)

**Comment by am-abudu** [comment]
这或许是一个程序错误，我将检查

**Comment by am-abudu** [comment, marked as answer]
See #491
